packages/generator/templates/app/src/app/auth/reset-password/page.tsx

```diff
@@ -0,0 +1,64 @@
+"use client"
+import {LabeledTextField} from "src/app/components/LabeledTextField"
+import {Form, FORM_ERROR} from "src/app/components/Form"
+import {ResetPassword} from "../validations"
+import resetPassword from "../mutations/resetPassword"
+import {BlitzPage} from "@blitzjs/next"
+import {useMutation} from "@blitzjs/rpc"
+import Link from "next/link"
+import {useSearchParams} from "next/navigation"
+
+const ResetPasswordPage: BlitzPage = () => {
+  const searchParams = useSearchParams()
+  const token = searchParams?.get("token")?.toString()
+  const [resetPasswordMutation, {isSuccess}] = useMutation(resetPassword)
+
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```

Same as above for the other page with a wrapper div